plugins/aggregators/starlark/testdata/merge.star

```diff
@@ -0,0 +1,25 @@
+# Example of a merge aggregator implemented with a starlark script.
+
+load('time.star', 'time')
+
+def add(cache, metric):
+    metrics = cache.get("metrics")
+    if metrics == None:
+        metrics = {}
+        cache["metrics"] = metrics
+        cache["ordered"] = []
+    m = metrics.get(metric)
+    if m == None:
+        m = deepcopy(metric)
+        metrics[metric] = m 
+        cache["ordered"].append(m)
+    else:
+        for k, v in metric.fields.items():
+            m.fields[k] = v
+
+def push(cache, accumulator):
+    accumulator.set_precision(time.nanosecond)
+    metrics = cache.get("ordered")
+    if metrics != None:
+        for m in metrics:
+             accumulator.add_metric(m)
```

Comment:
is ordered literally the same objects as the one in the metrics cache?

Reply:
It is the equivalent of the [Push function of the merge aggregator](https://github.com/influxdata/telegraf/blob/master/plugins/aggregators/merge/merge.go#L46-L48), it is only a simple rewrite, unless I miss something. Keep in mind that they are only here for testing purpose as POC.